user_tools/src/spark_rapids_pytools/resources/profiling-conf.yaml

```diff
@@ -1,5 +1,7 @@
 toolOutput:
   subFolder: rapids_4_spark_profile
+  textFormat:
+    log4jFileName: rapids_4_spark_profile_stderr.log
```

Comment:
QQ: Double checking that it is verified that the content of this file does not get included in the CI/CD tests that compare the output of 2 runs.

Reply:
Thanks for checking this. I updated the script in cicd to ignore this file.